pages/about/community.md

```diff
@@ -28,7 +28,7 @@ If you're having trouble, email [{{ site.uswds_email }}](mailto:{{ site.uswds_em
 There are several ways to contribute to USWDS. Participating in the community is one way to contribute. See [our contributing guidelines](https://github.com/uswds/uswds/blob/develop/CONTRIBUTING.md) for more about how to contribute back to our codebase. Also, please note that every submission we receive goes through a full design and code review.
 
 <p>
-  <a href="https://github.com/uswds/uswds" class="usa-button usa-button--outline width-card-lg margin-top-2">USWDS on GitHub</a>
+  <a href="https://github.com/uswds/uswds" class="usa-button site-button usa-button--outline">USWDS on GitHub</a>
```

Comment:
Note: The `site-button` class changes the font weight. I think it looks better to have the default bolder font on the outline button variant.

![image](https://github.com/uswds/uswds-site/assets/93996430/c41cea85-78fc-4412-8f1a-b22008eb68d9)

Reply:
Resolved in https://github.com/uswds/uswds-site/pull/2318/commits/024d4d1d26277dc8c23bcef9a6b1b21dd54a978e